### ISSUE TITLE
Full screen map: implement "subsetting mode" for when there is no overlay variable

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/web-eda#1497]
Requires: https://github.com/VEuPathDB/web-components/issues/421

Extend the `useMapMarkers` hook to perform filtered and unfiltered calls to `map-markers` and construct DonutMarkers (cumulative mode) or ChartMarkers in the standard pink/grey colour scheme.

At some point there will be a [legend](https://github.com/VEuPathDB/web-eda/issues/1496) with a radio selector for donut/chart markers.

**Comment by bobular** [comment]
No longer needed in SAM land
